### DISCUSSION TITLE
Using Eww to set wallpaper?

### DISCUSSION THREAD

**Discussion by AstralScribe** [elkowar/eww#1124] [Q&A]
I have just started reading the eww documentation and the dharmx example for the powermenu made me think of the possiblity of using eww for setting wallpaper instead of swww which I am currently using. Is this feasable, or is there a downside to doing this?

**Comment by MyGitHubBlueberry** [comment]
Swww has more options for wallpaper (e.g. switching animations). But you can use eww if you really want to)

**Comment by w-lfchen** [comment]
what exactly do you want to implement?
if you want to display a static wallpaper, i'd recommend swaybg for wayland/something similar for x11
